domain/removal/service/application.go

```diff
@@ -71,7 +71,7 @@ func (s *Service) RemoveApplication(
 	}
 
 	// Ensure the application is not alive.
-	unitUUIDs, machineUUIDs, err := s.modelState.EnsureApplicationNotAliveCascade(ctx, appUUID.String())
+	artifacts, err := s.modelState.EnsureApplicationNotAliveCascade(ctx, appUUID.String())
```

Comment:
You're English, which would make this "artefacts". :stuck_out_tongue:

Reply:
Wow! TIL!